build-logic/src/main/kotlin/com/ibm/wala/gradle/JavaCompileUsingEcj.kt

```diff
@@ -65,10 +65,18 @@ abstract class JavaCompileUsingEcj : JavaCompile() {
 
   @TaskAction
   protected override fun compile(inputs: InputChanges) {
+    val testArgs = options.allCompilerArgs
+    val argFile = createTempFile(prefix = "kotlinTemp", suffix = ".tmp")
+    val writer = java.io.PrintWriter(argFile)
+    for (testArg in testArgs) {
+      writer.print(testArg + " ")
```

Comment:
In my opinion, this file would be more readable (e.g., when debugging) if arguments were separated by line breaks instead of spaces. If we change `print` to `println` and remove the `+ " "`, does that still work correctly on all supported platforms? Or does something require that we use spaces instead of newlines?

Reply:
i tried a test, and it seems to want the arguments on a single line

Reply:
OK, thanks for giving that a try. I'm satisfied with this code change as originally proposed.